syft/pkg/cataloger/binary/classifier_cataloger_test.go

```diff
@@ -1394,6 +1394,65 @@ func Test_Cataloger_PositiveCases(t *testing.T) {
 		},
 	}
 
+	imageFixtureTests := []struct {
```

Comment:
could whole test should probably follow the patterns above, where it's just another entry like
```
		{
			logicalFixture: "ffmpeg-shared-libs/5.1.4/linux-amd64",
			expected: pkg.Package{
				Name:      "ffmpeg",
				Version:   "5.1.4",
				Type:      "binary",
				PURL:      "pkg:generic/ffmpeg@5.1.4",
				Locations: locations("ffmpeg"),
				Metadata:  metadata("ffmpeg-binary"),
			},
		},
```
?